packages/next/README.md

```diff
@@ -908,6 +912,51 @@ class MyLink extends React.Component {
 export default withRouter(MyLink)
 ```
 
+### API routes
```

Comment:
We use uppercase for starting letters in titles, this would also require an update with `doctoc`:

```suggestion
### API Routes
```